registry.json

```diff
@@ -668,5 +668,20 @@
  "telegram": "rabbitholepools",                                            
  "operatorName": "Adrem",                                
  "github": "adrem1"                                               
- } 
-] 
+ },
+ {
+ "ticker": "AFRIK",                                                      
+ "poolId": "fd16af916631bd29bf926d210da45d4aaa852242965ad06c124726f1", 
+ "social": {                                                           
+   "facebook": "", 
+   "twitter": "Cardano_Afrik",
+   "website": "https://cardano.afrikpool.org/",
+   "reddit": "",
+   "instagram": "",
+   "youtube": "https://www.youtube.com/channel/UC-MTKXd0S-nkDNlNlQd853Q"
+ },
+ "telegram": "cardano_afrik",
```

Comment:
Please use your personal Telegram handle :)

Reply:
Change it to my own telegram account :)